src/ops.rs

```diff
@@ -106,7 +106,13 @@ impl Sodg {
             .vertices
             .get_mut(&v)
             .context(format!("Can't find ν{v}"))?;
+        if vtx.posted {
+            return Err(anyhow!(
+                "The vertex isn't available, #put() was called earlier"
```

Comment:
@l3r8yJ it's better to say "the vertex is _full_". Also, please see how we format other error messages in other methods. We usually try to provide as much context as possible.